src/main/java/org/folio/inventory/consortium/handlers/MarcInstanceSharingHandlerImpl.java

```diff
@@ -193,25 +194,30 @@ Future<Record> getSourceMARCByInstanceId(String instanceId, String sourceTenant,
     return promise.future();
   }
 
-  Future<String> deleteSourceRecordByInstanceId(String recordId, String instanceId, String tenantId, SourceStorageRecordsClient client) {
-    LOGGER.info("deleteSourceRecordByInstanceId :: Delete source record with recordId={} for instance by InstanceId={} from tenant {}",
+  Future<String> deleteSourceRecordByRecordId(String recordId, String instanceId, String tenantId, SourceStorageRecordsClient client) {
+    LOGGER.info("deleteSourceRecordByRecordId :: Delete source record with recordId={} for instance by InstanceId={} from tenant {}",
       recordId, instanceId, tenantId);
     Promise<String> promise = Promise.promise();
     client.deleteSourceStorageRecordsById(recordId).onComplete(responseResult -> {
       try {
         if (responseResult.failed()) {
-          LOGGER.error("deleteSourceRecordByInstanceId:: Error deleting source record with recordId={} by InstanceId={} from tenant {}",
+          LOGGER.error("deleteSourceRecordByRecordId:: Error deleting source record with recordId={} by InstanceId={} from tenant {}",
             recordId, instanceId, tenantId, responseResult.cause());
           promise.fail(responseResult.cause());
+        } else if(responseResult.result().statusCode() != SC_NO_CONTENT) {
+          String msg = format("Error deleting source record with recordId=%s by InstanceId=%s from tenant %s, responseStatus=%s, body=%s",
```

Comment:
How does this change, solve multiple matches in SRS? is it supposed to solve multiple matches?

Reply:
The PR changes are aimed at correcting the log message in accordance with the response to the request for record deletion. The lack of permission itself, which eventually causes multiple matching, has been fixed in https://github.com/folio-org/mod-consortia/pull/129